x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -172,9 +174,9 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
     if (id === TimelineId.active) {
       refetchQuery([timelineQuery]);
     } else {
-      refetchQuery(globalQuery);
+      refetchQuery(globalQueries);
```

Comment:
This was renamed from `globalQuery` to `globalQueries` because it's really a collection of queries, (not a single query)